cobalt/h5vcc/h5vcc_settings.cc

```diff
@@ -81,22 +80,6 @@ bool H5vccSettings::Set(const std::string& name, SetValueType value) const {
     return true;
   }
 
-  if (name.compare(kClientHintHeaders) == 0 && value.IsType<int32>()) {
```

Comment:
Will deleting this logic break any web clients that try to set this value? I see it can be the difference between returning true and false here.